File: Iniciante/URI 1012.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');

var line = lines.shift().split(' ');
var a = parseFloat(line[0]);
var b = parseFloat(line[1]);
var c = parseFloat(line[2]);

var triangulo = a * c / 2.0;
var circulo = 3.14159 * (c * c);
var trapezio = ((a + b) * c) / 2;
var quadrado = b * b;
var retangulo = a * b;

console.log('TRIANGULO: ' + triangulo.toFixed(3));
console.log('CIRCULO: ' + circulo.toFixed(3));
console.log('TRAPEZIO: ' + trapezio.toFixed(3));
console.log('QUADRADO: ' + quadrado.toFixed(3));
console.log('RETANGULO: ' + retangulo.toFixed(3));

File: Iniciante/URI 1012.clj
(require '[clojure.string :as str])

(def line (str/split (read-line) #" "))
(def a (Float/parseFloat (get line 0)))
(def b (Float/parseFloat (get line 1)))
(def c (Float/parseFloat (get line 2)))

(def triangulo (/ (* a c) 2.0))
(def circulo (* 3.14159 (* c c)))
(def trapezio (/ (* (+ a b) c) 2.0))
(def quadrado (* b b))
(def retangulo (* a b))

(printf "TRIANGULO: %.3f\n" triangulo)
(printf "CIRCULO: %.3f\n" circulo)
(printf "TRAPEZIO: %.3f\n" trapezio)
(printf "QUADRADO: %.3f\n" quadrado)
(printf "RETANGULO: %.3f\n" retangulo)